src/metric/PopulationSimilaritySampleMetric.py

```diff
@@ -54,6 +55,7 @@ def calc(self, ids: pd.Index, **parameters: Any) -> np.ndarray:
 
         Args:
             ids (pd.Index): The indices of the samples to calculate for.
+            filter_bit (int): Filter bit to filter the population to calculate similarity against.
```

Comment:
a bit confusing maybe, think about formulation and give context as to why this is here